.github/workflows/Deploy.yml

```diff
@@ -29,6 +29,9 @@ jobs:
       # Setup the environment
       - run: 'json -I -f packages/svelte-vscode/package.json -e "this.version=\`99.0.0\`"'
       - run: 'json -I -f packages/svelte-vscode/package.json -e "this.preview=true"'
+      - run: 'json -I -f packages/svelte-vscode/package.json -e "this.name=\`svelte-vscode-nightly\`"'
+      - run: 'json -I -f packages/svelte-vscode/package.json -e "this.icon=\`icons/logo-nightly.png\`"'
+      - run: 'json -I -f packages/svelte-vscode/package.json -e "this.displayName=\`Svelte Language Tools Nightly Builds\`"'
```

Comment:
Bikeshedding#2: I think we should name the nightly almost the same as the stable one, just with a Nightly at the end. So for example `Svelte for VSCode (Nightly)` or `Svelte (Official, Nightly)`.

Reply:
I'm not against it, but I choose this originally _because_ it's a bit more technically intimidating - people shouldn't really need to use the daily build very often and giving it the more unfriendly name was to discourage folks. We do the same for [TS' nightly builds](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-typescript-next) in vscode

Reply:
I get the intimidating part, but I still feel like people should at least be able to discover it easily. What about `<Original name> Unstable Nightly Builds` instead? The `Unstable` will trigger most people to not use it in their day-to-day-work.

Reply:
Works for me